xendit-java-lib/src/main/java/com/xendit/model/Disbursement.java

```diff
@@ -49,6 +49,24 @@ public class Disbursement {
   @SerializedName("failure_code")
```

Comment:
hey @hglohika instead of modifying this file, can you create a different model instead for PH?
`DisbursementPH.java`
then rename the old one to `DisbursementIDR.java`

Reply:
15a370ad569f9d5ff575f225778dd4ce9d7118af